test/SIL/OwnershipVerifier/use_verifier.sil

```diff
@@ -1383,20 +1359,6 @@ bb3(%fUnknown : @owned $@callee_owned () -> ()):
   return %9999 : $()
 }
 
-sil [ossa] @unowned_to_ref_is_unowned_instant_use : $@convention(thin) (@guaranteed Builtin.NativeObject) -> Builtin.NativeObject {
```

Comment:
These ver specifically testing the unowned return type which is not supported. I think these or OK to remove.